docs/architecture/databaseUtil.md

```diff
@@ -0,0 +1,87 @@
+# DatabaseUtil
+
+The [DatabaseUtil](../../src/utils/databaseUtil.ts) abstracts
+writing and reading to the file system,
+as well as parsing and serializing the game data.
+
+For now it is assumed that there is only one save file.
+
+## Delete
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant A as Actor
+  participant D as DatabaseUtil
+  participant F as File System
+
+  A ->> D: delete()
+  D ->> F: unlink()
+  D ->> D: error: unknown | null
```

Comment:
Should this just be `error: null`? Since error is only set to `unknown` when there is a error.

Reply:
This was left over from before I did the error handling. Responses should only be in the `alt` and `else` blocks

Reply:
Removed